src/main/java/io/github/mountainrange/mule/GameManager.java

```diff
@@ -216,10 +227,14 @@ private boolean allBoughtLand() {
 		return true;
 	}
 
-	private void setLabels() {
+	public void setLabels() {
 		turnLabel.setText(playerList.get(turnOrder.get(currentPlayer)).getName() + "'s Turn " + timeLeft);
 		resourceLabel.setText(playerList.get(turnOrder.get(currentPlayer)).getName() + "'s Money: "
-				+ playerList.get(turnOrder.get(currentPlayer)).getMoney() + " Energy: ####");
+				+ playerList.get(turnOrder.get(currentPlayer)).getMoney() + " F: "
+				+ playerList.get(turnOrder.get(currentPlayer)).stockOf(ResourceType.FOOD) + " E: "
+				+ playerList.get(turnOrder.get(currentPlayer)).stockOf(ResourceType.ENERGY) + " S: "
+				+ playerList.get(turnOrder.get(currentPlayer)).stockOf(ResourceType.SMITHORE) + " C: "
+				+ playerList.get(turnOrder.get(currentPlayer)).stockOf(ResourceType.CRYSTITE));
```

Comment:
Consider using string formatting rather than string concatenation, and refactoring this to include some temporary variables, so that it looks nicer.